source/converter/contracts/Converter.sol

```diff
@@ -0,0 +1,252 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "./TokenPaymentSplitter.sol";
+
+interface IUniswapV2Router02 {
+  function swapExactTokensForTokensSupportingFeeOnTransferTokens(
+    uint256 amountIn,
+    uint256 amountOutMin,
+    address[] calldata path,
+    address to,
+    uint256 deadline
+  ) external;
+}
+
+contract Converter is Ownable, ReentrancyGuard, TokenPaymentSplitter {
+  using SafeMath for uint256;
+  using SafeERC20 for IERC20;
+
+  address public constant WETH =
+    address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+
+  address public swapToToken;
+
+  address public immutable uniRouter;
+
+  uint256 public triggerFee;
+
+  mapping(address => address[]) private tokenPathMapping;
+
+  address[] public currTokens;
```

Comment:
This was originally part of the first version of the drain function, and is no longer needed/used, so going to remove it.

Reply:
Actually wait... it was a way to quickly find what are all the tokens that have been sent to this contract, so that if you need to use the drainAll function then you can call this to get an array, and then manually pass that array as a parameter when calling the drainAll function.
Otherwise if you need to drainAll then you could look up manually on EtherScan to see what are all the tokens in the contract and create your own array from that.
If drainAll is probably going to be rarely called, then happy to take this out if not necessary.

Reply:
I removed it, as it isn't really needed.